Sources/PubNubChatComponents/Controllers/ComponentViewController.swift

```diff
@@ -125,4 +127,22 @@ open class CollectionViewComponent: ComponentViewController  {
     collectionView.leadingAnchor.constraint(equalTo: view.leadingAnchor, constant: 0).isActive = true
     collectionView.trailingAnchor.constraint(equalTo: view.trailingAnchor, constant: 0).isActive = true
   }
+  
+  open func configureGestureRecognizers() {
+    let longPressGesture = UILongPressGestureRecognizer(target: self, action: #selector(onLongPressGestureRecognized(gesture:)))
+    let swipeGestureRecognizer = UISwipeGestureRecognizer(target: self, action: #selector(onSwipeGestureRecognized(gesture:)))
+    
+    collectionView.addGestureRecognizer(longPressGesture)
+    collectionView.addGestureRecognizer(swipeGestureRecognizer)
```

Comment:
Not that you need to implement it this way, but you can probably make a Combine Publisher/Subscriber for Gesture Events that follow this pattern for [UIControl.Events](https://github.com/pubnub/chat-components-ios/blob/master/Sources/PubNubChatComponents/Extensions/Combine%2BPubNub.swift#L55-L100)

Reply:
Do we use gesture recognizers anywhere else? Anyway, I like this idea and we could create a ticket to make a Publisher and Subscriber for that